gtda/images/filtration.py

```diff
@@ -0,0 +1,160 @@
+"""Binary image filtration."""
+# License: GNU AGPLv3
+
+import numbers
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from joblib import Parallel, delayed, effective_n_jobs
+from sklearn.utils import gen_even_slices
+from sklearn.utils.validation import check_is_fitted, check_array
+from ..utils._docs import adapt_fit_transform_docs
+from ..utils.validation import validate_params
+
+
+@adapt_fit_transform_docs
+class HeightFiltration(BaseEstimator, TransformerMixin):
+    """Transformer returning a collection of grayscale images
+    from a collection of 2D or 3D binary images.
+
+    The height filtration assigns to each activated pixel of an image a pixel
```

Comment:
Yes, an hyperplane is defined by a vector as everything perpendicular to it. You also need a point in space. It's hard to explain it otherwise. Let's discuss that face 2 face and try to find a better explanation.

Reply:
Added to kanban